js/ctrls/main.js

```diff
@@ -398,6 +398,10 @@ function(
 				animCollapsed: true,
 				files: [],
 			};
+			if (d.verifiedLength)
+				ctx.verifiedLength = d.verifiedLength;
```

Comment:
Will `verifiedLength` still exist after its all verified, and the real download starts? 

Normally my verifying time is low so cant test this for all edge cases ..